src/ui/app/tabs/trezorTx.jsx

```diff
@@ -89,11 +92,14 @@ const App = () => {
 
 const root = createRoot(window.document.querySelector(`#${TAB.trezorTx}`));
 root.render(
-  <Main>
-    <Router>
-      <App />
-    </Router>
-  </Main>
+  <AnalyticsProvider view={ExtensionViews.Extended}>
```

Comment:
@DominikGuzei do we want to be more specific? e.g: Trezor

Reply:
For now i think it's fine, and its a decision Danilo and Rhys have to make which properties we want to track

Reply:
Let's not bother with this detail, it's not that relevant